code-submissions-clustering-core/src/main/proto/SubmissionsGraph.proto

```diff
@@ -5,10 +5,10 @@ option java_multiple_files = true;
 option java_outer_classname = "SubmissionsGraphProtos";
 
 message ProtoSubmissionsNode {
-  int32 id = 1;
-  string code = 2;
-  int32 stepId = 3;
-  repeated int32 idList = 4;
+  int32 id = 4;
+  string code = 1;
+  int32 stepId = 2;
+  repeated int32 idList = 3;
```

Comment:
please, reorder it:
string code = 1;
int32 stepId = 2;
repeated int32 idList = 3;
int32 id = 4;